tests/cc13b/input.dat

```diff
@@ -0,0 +1,50 @@
+#! Tests RHF CCSD(T)gradients
+memory 500 mb
+
+molecule h2o {
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+   reference rhf
+   basis cc-pvdz
+   r_convergence 13
+   e_convergence 13
+   d_convergence 13
+}
+
+analytic, wfn = gradient('ccsd(t)', return_wfn=True)
+
+set findif points 5
+fin_diff, wfn = gradient('ccsd(t)', dertype=0, return_wfn=True)
+
+compare_matrices(analytic, fin_diff, 10, "RHF-CCSD(T) gradient test") #TEST
```

Comment:
Concur with @amjames that it would be good to have a quick rhf-ccsd(t) grad in quicktests. In the cfour tests, you'll find [some gradients](https://github.com/psi4/psi4/blob/master/tests/cfour/opt-rhf-ccsd_t_/output.ref#L1024-L1034) if you'd like to compare with that.